<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Andy Styles</author>
 <class>MainWindowDialog</class>
 <widget class="QMainWindow" name="MainWindowDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Q3270</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="terminalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Session</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionSessionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecentSessions">
     <property name="title">
      <string>Recent Sessions</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="menuRecentSessions"/>
    <addaction name="actionSave_Session"/>
    <addaction name="actionSave_SessionAs"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Q3270"/>
    <addaction name="actionConnection_Information"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionManage_Sessions"/>
    <addaction name="actionManage_Auto_Sart_Sessions"/>
    <addaction name="actionKeyboardThemes"/>
    <addaction name="actionColourThemes"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnect"/>
   <addaction name="menuManage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Session"/>
   <addaction name="actionSave_Session"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionSessionPreferences"/>
   <addaction name="separator"/>
   <addaction name="actionManage_Sessions"/>
   <addaction name="actionManage_Auto_Sart_Sessions"/>
   <addaction name="actionKeyboardThemes"/>
   <addaction name="actionColourThemes"/>
  </widget>
  <action name="actionConnect">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/Q3270.qrc">
     <normaloff>:/Icons/connect.svg</normaloff>:/Icons/connect.svg</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionSessionPreferences">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/Q3270.qrc">
     <normaloff>:/Icons/settings.svg</normaloff>:/Icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/Q3270.qrc">
     <normaloff>:/Icons/disconnect.svg</normaloff>:/Icons/disconnect.svg</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New Window...</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reconnect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_Q3270">
   <property name="text">
    <string>About Q3270...</string>
   </property>
  </action>
  <action name="actionKeyboardThemes">
   <property name="icon">
    <iconset resource="../resources/Q3270.qrc">
     <normaloff>:/Icons/keyboard.svg</normaloff>:/Icons/keyboard.svg</iconset>
   </property>
   <property name="text">
    <string>Keyboard Maps...</string>
   </property>
  </action>
  <action name="actionColourThemes">
   <property name="icon">
    <iconset resource="../resources/Q3270.qrc">
     <normaloff>:/Icons/palette.svg</normaloff>:/Icons/palette.svg</iconset>
   </property>
   <property name="text">
    <string>Colour Themes...</string>
   </property>
  </action>
  <action name="actionSave_SessionAs">
   <property name="text">
    <string>Save Session As...</string>
   </property>
  </action>
  <action name="actionOpen_Session">
   <property name="icon">
    <iconset resource="../resources/Q3270.qrc">
     <normaloff>:/Icons/open.svg</normaloff>:/Icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>Open Session...</string>
   </property>
  </action>
  <action name="actionOpen_Duplicate_Session">
   <property name="text">
    <string>Open Duplicate Session</string>
   </property>
  </action>
  <action name="actionManage_Sessions">
   <property name="icon">
    <iconset resource="../resources/Q3270.qrc">
     <normaloff>:/Icons/database.svg</normaloff>:/Icons/database.svg</iconset>
   </property>
   <property name="text">
    <string>Sessions...</string>
   </property>
  </action>
  <action name="actionSave_Session">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/Q3270.qrc">
     <normaloff>:/Icons/floppy.svg</normaloff>:/Icons/floppy.svg</iconset>
   </property>
   <property name="text">
    <string>Save Session</string>
   </property>
  </action>
  <action name="actionConnection_Information">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About Connection...</string>
   </property>
  </action>
  <action name="actionManage_Auto_Sart_Sessions">
   <property name="icon">
    <iconset resource="../resources/Q3270.qrc">
     <normaloff>:/Icons/database-gear.svg</normaloff>:/Icons/database-gear.svg</iconset>
   </property>
   <property name="text">
    <string>Auto Start Sessions...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/Q3270.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>menuConnect()</slot>
  <slot>menuQuit()</slot>
  <slot>menuSessionPreferences()</slot>
  <slot>menuSetFont()</slot>
  <slot>menuDisconnect()</slot>
  <slot>menuNew()</slot>
  <slot>menuTabbedView(bool)</slot>
  <slot>menuReconnect()</slot>
  <slot>menuAbout()</slot>
  <slot>menuColourTheme()</slot>
  <slot>menuSaveSessionAs()</slot>
  <slot>menuOpenSession()</slot>
  <slot>menuKeyboardTheme()</slot>
  <slot>menuDuplicate()</slot>
  <slot>menuManageSessions()</slot>
  <slot>menuSaveSession()</slot>
  <slot>menuAboutConnection()</slot>
 </slots>
</ui>
